### PR TITLE
Use azd feature

### PR DESCRIPTION
**Pull request by pamelafox**
This PR updates the dev container to use the azd feature.

### CODE REVIEW

File: {{cookiecutter.__src_folder_name}}/root/docker-compose.yml [hunk 1]
```diff
@@ -18,7 +19,6 @@ services:
       timeout: 5s
       retries: 5
   app:
-    platform: linux/amd64
```
Comment (pamelafox) [inline {{cookiecutter.__src_folder_name}}/root/docker-compose.yml:21]: This was the line that caused us trouble before

File: {{cookiecutter.__src_folder_name}}/root/docker-compose.yml [hunk 2]
```diff
@@ -5,10 +5,10 @@ services:
     image: postgres:14
 
     environment:
-      DBSERVER_USER: postgres
-      DBSERVER_DB: relecloud
-      DBSERVER_HOST: db
-      DBSERVER_PORT: 5432
```
Comment (pamelafox) [inline {{cookiecutter.__src_folder_name}}/root/docker-compose.yml:11]: This seemed to get the wrong env variable names at some point. Its also redundant with whats in the dev compose, so maybe could be rm'ed here or there.

File: {{cookiecutter.__src_folder_name}}/entrypoint.sh
Comment (kjaymiller) [inline {{cookiecutter.__src_folder_name}}/entrypoint.sh:1]: if the naming standard is usually `app` or `applications` do we want to deviate with `fastapi_app`

Comment (kjaymiller) [reply {{cookiecutter.__src_folder_name}}/entrypoint.sh:1]: Ah I see you went with the "make it a package so usage is easier down the road.

File: {{cookiecutter.__src_folder_name}}/fastapi/fastapi_app/app.py [hunk 3]
```diff
@@ -1,20 +1,18 @@
...
-    engine
-)
+
+from .models import Cruise, Destination, InfoRequest, engine
 
 app = FastAPI()
-app.mount('/mount', StaticFiles(directory="static"), name="static")
-templates = Jinja2Templates(directory="templates")
+current_path = pathlib.Path(__file__).parent
+app.mount('/mount', StaticFiles(directory=current_path.parent / "static"), name="static")
```
Comment (kjaymiller) [inline {{cookiecutter.__src_folder_name}}/fastapi/fastapi_app/app.py:14]: Surprised that `'/mount'` didn't become `"mount"`

File: {{cookiecutter.__src_folder_name}}/requirements.txt
Comment (kjaymiller) [inline {{cookiecutter.__src_folder_name}}/requirements.txt:1]: With django being the only version that isn't packaged should we look at the feasibility of packaging django?

File: {{cookiecutter.__src_folder_name}}/root/.devcontainer/Dockerfile_dev
Comment (kjaymiller) [inline {{cookiecutter.__src_folder_name}}/root/.devcontainer/Dockerfile_dev:1]: RE: Removing xdg-utils - Was it installing something that was breaking things?

File: {{cookiecutter.__src_folder_name}}/root/docker-compose.yml (hunk 2 above)
Comment (kjaymiller) [reply {{cookiecutter.__src_folder_name}}/root/docker-compose.yml:11]: Let's discuss this early next week

**Review by kjaymiller — CHANGES_REQUESTED** [review]
I have a couple questions we can chat about early next week but all in all great! Looks good.

File: {{cookiecutter.__src_folder_name}}/root/.devcontainer/Dockerfile_dev
Comment (pamelafox) [reply {{cookiecutter.__src_folder_name}}/root/.devcontainer/Dockerfile_dev:1]: The azd feature installs xdg-utils for us now

File: {{cookiecutter.__src_folder_name}}/fastapi/fastapi_app/app.py (hunk 3 above)
Comment (pamelafox) [reply {{cookiecutter.__src_folder_name}}/fastapi/fastapi_app/app.py]: You're right, I hadn't pushed the latest reformats

File: {{cookiecutter.__src_folder_name}}/entrypoint.sh
Comment (pamelafox) [reply {{cookiecutter.__src_folder_name}}/entrypoint.sh:1]: Yeah, I wanted to be able to run everything from the root, and that was easiest way.

File: {{cookiecutter.__src_folder_name}}/requirements.txt
Comment (pamelafox) [reply {{cookiecutter.__src_folder_name}}/requirements.txt:1]: Maybe, I'm going to test django now. It's managed so differently, with manage.py and the project env variable, that packaging may not be needed / desirable.

**Review by kjaymiller — APPROVED** [review]

**Pull request merged**